examples/README.md

```diff
@@ -1,4 +1,4 @@
-# pytket examples
+# Contributing new notebooks
```

Comment:
This is a good change to reflect what the content of this file is, but it makes it clear it's not really a README file at all. I'd suggest:
* renaming this file to `CONTRIBUTING.md` or something like that, removing the "See the pytket examples" paragraph, and including CONTRIBUTING.md in the docs.
* adding a new `examples/README.md` file which has the "See the pytket examples" paragraph and a link to CONTRIBUTING.md, and do not include that README in the docs.

I can make this change if it's easier to explain that way?

Reply:
Also, if you do go for `CONTRIBUTING.md` then I'd suggest it be in its own section in the table of contents, rather than bundled into "Others".

Reply:
Yeah adding a `CONTRIBUTING.md` sounds like a good idea. Thanks.